Resources/Prototypes/_Echo/Entities/Objects/Fun/centcom_pai.yml

```diff
@@ -0,0 +1,66 @@
+- type: entity
+  parent: PersonalAI
+  id: CentComPersonalAI
+  name: CentCom personal ai device
+  description: Your Central Command pal who might be fun to be with!
+  components:
+  - type: PAI
+    namePrefix: pai-system-name-prefix-centcom
+    paiName: pai-system-name-centcom
+    paiNameRaw: pai-system-name-raw-centcom
+  - type: PAICentCom
+  - type: Sprite
+    sprite: _Echo/Objects/Fun/centcom_pai.rsi
+    layers:
+    - state: centcom-pai-base
+    - state: centcom-pai-off-overlay
+      shader: unshaded
+      map: [ "screen" ]
+  - type: ToggleableGhostRole
+    roleName: pai-system-role-name-centcom
+    roleDescription: pai-system-role-description-centcom
+    roleRules: ghost-role-information-familiar-rules
+  - type: Access
+    groups:
+    - AllAccess
+  - type: Tag
+    tags:
+    - DoorBumpOpener
+  - type: IntrinsicRadioTransmitter
+    channels:
+    - Binary
+    - CentCom
+    - Command
+  - type: IntrinsicRadioFromKeys
+    permanentChannels:
+    - Binary
+    - CentCom
+    - Command
+  - type: ActiveRadio
+    channels:
+      - Binary
+      - CentCom
+      - Command
+  - type: GenericVisualizer
+    visuals:
+      enum.ToggleableGhostRoleVisuals.Status:
+        screen:
+          Off: { state: centcom-pai-off-overlay }
+          Searching: { state: centcom-pai-searching-overlay }
+          On: { state: centcom-pai-on-overlay }
+
+- type: entity
+  id: ActionPAIElectrocute
+  name: Electrocute
+  description: Electrocute whoever is near you.
+  components:
+  - type: InstantAction
+    checkCanInteract: false
+    checkConsciousness: false
+    itemIconStyle: BigAction
+    priority: 20
+    icon:
+      sprite: _Echo/Interface/Actions/pai-shock.rsi
+      state: pai-shock
+    event: !type:PAIElectrocuteActionEvent
+    useDelay: 60
```

Comment:
I think 15 seconds should be okay. If someone really keeps picking up the pAI to try to kill it it should be able to defend itself reasonably well with that.